src/frontend/js/components/UserLogin/index.tsx

```diff
@@ -29,49 +28,31 @@ const messages = defineMessages({
     description: 'Accessibility text for the spinner in the login area.',
     id: 'components.UserLogin.spinnerText',
   },
+  account_settings: {
```

Comment:
not sure how it is better than the menus configurable via the backend settings? I'm afraid it will be hard for someone to add an item to this menu or customize it?

Reply:
Good point, I keep the previous mechanism to set user profile urls directly through settings.